app/public/templates/scene.html

```diff
@@ -81,11 +82,22 @@
         <a-entity
           id="primaryCamera"
           camera
+          fov="80"
           {% if scene_dict["camera_properties"]["look_controls"] %}
             look-controls
           {% end %}
           position="0 {{ scene_dict['camera_properties']['position'][1] }} 0"
-        ></a-entity>
+        >
+        <!--   Hints    -->
+        <a-entity
+          id="hint-button"
+          class="link"
+          hint-button='jsonData: {"newHintImg": "/static/img/lightOn.png", "hintImg": "/static/img/lightOff.png", "width": 0.08, "height": 0.08}'
+          animation__onclick="property: position; dur: 0; startEvents: click"
+          animation-on-click-run='blackboardData: {"jsonData": {"text": {% raw json.dumps(scene_dict["hints"]) %}, "currPosition": 0}, "componentType": "text-pane"}'
```

Comment:
Should we pass back a default value for "text" when there are no hints (ie. empty array)? Something like "no hints at the moment" would make sense to me.

Reply:
Yes, this will be done on the backend once designers have the right lingo ironed out. The default will be updated here: https://github.com/uwblueprint/cdc-backend/blob/master/app/models/scene.py#L18. For now, that's why I have put `generic hint for all scenes` as the first element in the mockdata. There will always be at least one hint.